src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1250,12 +1250,15 @@ void PolicyHandler::OnAllowSDLFunctionalityNotification(
     if (is_allowed) {
       // Send HMI status notification to mobile
       // Put application in full
-      AudioStreamingState::eType state = app->is_audio()
+      AudioStreamingState::eType audio_state = app->IsAudioApplication()
```

Comment:
@LitvinenkoIra would be nice to add **const** AudioStreamingState::eType audio_state = ...